robot/EDA/resources/locators_50.py

```diff
@@ -25,6 +25,12 @@
         "tab": "//div[@class='uiTabBar']/ul[@class='tabs__nav']/li[contains(@class,'uiTabItem')]/a[@class='tabHeader']/span[contains(text(), '{}')]",
         "spl-tab": "//div[@class='slds-tabs_default']//ul[@class='slds-tabs_default__nav']/li[contains(@class,'slds-tabs_default__item')]/a[text()= '{}']",
     },
+    "eda_setup": {
+        "custom_settings": "//a[text()='{}']",
```

Comment:
since this is a very common locator, you might want to move it to the outer dictionary. Just a suggestion

Reply:
Already had a similar locator in eda dictionary and made use of it and this is removed now

Reply:
I still see the locator